python/tests/README.md

```diff
@@ -4,5 +4,5 @@ pyValidator.sh creates a directory for Examples.py to write the generated exampl
 If all files are valid, the directory will be deleted. If at least one file fails validation, the directory will remain.
 
 * Before running, be sure to mvn install AIF.
-* To execute the script, run ./pyValidator.sh
```

Comment:
Add a line to run the command "pip install -e . ". I ran the pip install -r requirements.txt and it didn't install AIF. Also, please clarify the instructions on line #6. It currently looks like the command to run is "mvn install AIF" which may cause some confusion.

Reply:
Good point. I redid most of the README.md. Let me know what you think

Reply:
One more minor change. Please specify that the script command should be ran in the python/test directory just for clarification.

Reply:
Removed java installation instructions from README.md and referenced /java/README.md